packages/tracing/src/browser/metrics.ts

```diff
@@ -282,14 +282,14 @@ export class MetricsInstrumentation {
 
 /** Instrument navigation entries */
 function addNavigationSpans(transaction: Transaction, entry: Record<string, any>, timeOrigin: number): void {
-  addPerformanceNavigationTiming(transaction, entry, 'unloadEvent', timeOrigin);
-  addPerformanceNavigationTiming(transaction, entry, 'redirect', timeOrigin);
-  addPerformanceNavigationTiming(transaction, entry, 'domContentLoadedEvent', timeOrigin);
-  addPerformanceNavigationTiming(transaction, entry, 'loadEvent', timeOrigin);
-  addPerformanceNavigationTiming(transaction, entry, 'connect', timeOrigin);
-  addPerformanceNavigationTiming(transaction, entry, 'secureConnection', timeOrigin, 'connectEnd');
-  addPerformanceNavigationTiming(transaction, entry, 'fetch', timeOrigin, 'domainLookupStart');
-  addPerformanceNavigationTiming(transaction, entry, 'domainLookup', timeOrigin);
+  addPerformanceNavigationTiming({ transaction, entry, event: 'unloadEvent', timeOrigin });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'redirect', timeOrigin });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'domContentLoadedEvent', timeOrigin });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'loadEvent', timeOrigin });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'connect', timeOrigin });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'secureConnection', timeOrigin, eventEnd: 'connectEnd', description: 'secureConnection (TLS)' });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'fetch', timeOrigin, eventEnd: 'domainLookupStart', description: 'cache' });
+  addPerformanceNavigationTiming({ transaction, entry, event: 'domainLookup', timeOrigin });
```

Comment:
As per PR description, isn't this missing a `description: 'DNS'`?

Reply:
I've added this back in.